android/app/src/main/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "Email": "dev@fossasia.org",
   "App_Name": "Open Event",
-  "Api_Link": "https://eventyay.com/api/v1/events/6/"
+  "Api_Link": "https://api.eventyay.com/v1/events/69/"
```

Comment:
@Shailesh351 I _am_ actually talking about testing ... Like I said ... the server is not dependable. The db can get flushed. And if it does, your tests and CI will fail here.